### ISSUE TITLE
feature request: rhn_channel, and other rhn_* modules, should have validate_certs parameter

### ISSUE THREAD

**Issue by ansibot** [ansible/ansible#29649]
From @snemir2 on 2016-09-15T20:21:40Z

##### ISSUE TYPE

- Feature Idea
##### COMPONENT NAME

rhn_channel
rhn_*
##### ANSIBLE VERSION

```

[snemirovsky@SN-WS-Fedora24 patch]$ ansible --version
[WARNING]: log file at /var/log/ansible.log is not writeable and we cannot create it, aborting

ansible 2.1.1.0
  config file = /ansible/conf/ansible.cfg
  configured module search path = Default w/o overrides

```
##### STEPS TO REPRODUCE

```

```

##### EXPECTED RESULTS

excected my code to work 
##### ACTUAL RESULTS

```
TASK [RHN_CHANNEL_configure : Configure Channel on Satellite] ******************
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ssl.SSLError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:590)
```

Copied from original issue: ansible/ansible-modules-core#4876

**Comment by ansibot** [comment]
From @snemir2 on 2016-09-15T20:21:40Z

i wish there was validate_cert flag same as for uri module....

**Comment by ansibot** [comment]
From @ansibot on 2016-09-15T20:21:40Z

@vincentvdk, @alikins, ping. This issue is waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by biomassives** [comment]
This message was created automatically by mail delivery software.

A message that you sent could not be delivered to one or more of its
recipients. This is a temporary error. The following address(es) deferred:

  acmeideal@gmail.com
    Domain biomassiv.es has exceeded the max emails per hour (120/100 (120%)) allowed.  Message will be reattempted later

------- This is a copy of the message, including all the headers. ------
Received: from github-smtp2-ext8.iad.github.net ([192.30.252.199]:38143 helo=github-smtp2a-ext-cp1-prd.iad.github.net)
	by chi-server32.websitehostserver.net with esmtps (TLSv1.2:ECDHE-RSA-AES256-GCM-SHA384:256)
	(Exim 4.89)
	(envelope-from <noreply@github.com>)
	id 1drSvT-002T6L-Cb
	for greg@biomassiv.es; Mon, 11 Sep 2017 12:55:15 -0500
Date: Mon, 11 Sep 2017 10:54:29 -0700
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=github.com;
	s=pf2014; t=1505152474;
	bh=J8ISl7aFBKgZ9WG9QhhYDBNSnDdrM9FyQSjMdxrJnBI=;
	h=From:Reply-To:To:Cc:In-Reply-To:References:Subject:List-ID:
	 List-Archive:List-Post:List-Unsubscribe:From;
	b=VbRLJjGcJCosAFQrdiHbP0eq60zD234HbkRt/nK13q0qJN7qWWsU9UziSjRBkm03+
	 4hV/WubIFCohAdAOD0Oh0FJ1VquFMmsk6hWLehWdfwC9SPjNOxwHWK96Bj+Ap3n8LA
	 topjS4lwKwPGUu+zd5AdujfZbsmTsBrMEzc/dWMw=

**Comment by biomassives** [comment]
This message was created automatically by mail delivery software.

A message that you sent could not be delivered to one or more of its
recipients. This is a temporary error. The following address(es) deferred:

  acmeideal@gmail.com
    Domain biomassiv.es has exceeded the max emails per hour (123/100 (123%)) allowed.  Message will be reattempted later

------- This is a copy of the message, including all the headers. ------
------ The body of the message is 5586 characters long; only the first
------ 5000 or so are included here.
Received: from github-smtp2-ext2.iad.github.net ([192.30.252.193]:60887 helo=github-smtp2b-ext-cp1-prd.iad.github.net)
	by chi-server32.websitehostserver.net with esmtps (TLSv1.2:ECDHE-RSA-AES256-GCM-SHA384:256)
	(Exim 4.89)
	(envelope-from <noreply@github.com>)
	id 1drSvZ-002T9T-7e
	for greg@biomassiv.es; Mon, 11 Sep 2017 12:55:21 -0500
Date: Mon, 11 Sep 2017 10:54:40 -0700
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=github.com;
	s=pf2014; t=1505152480;
	bh=NCDWCUADdcd57s5R4OafN+2g1YIylcifYpxD2bwbwJI=;
	h=From:Reply-To:To:Cc:Subject:List-ID:List-Archive:List-Post:
	 List-Unsubscribe:From;
	b=DcHeTIcorBHQfILxivyBuGwhbYM0Qv0bhC5G73VKxJOr16OxbApvHDt5E6OpSD3II
	 blw/UNSMtbJzVmWIo70lUurF6YCy43Zj+ME2170Y/wBnXKfHbEKy4JPlOf6RX7co4H
	 n+IHS303ert2jWLiGca9RzzrMs42j2OxG+n1j5dI=

**Comment by ansibot** [comment]
Thank you very much for your interest in Ansible. Ansible has migrated much of the content into separate repositories to allow for more rapid, independent development. We are closing this issue/PR because this content has been moved to one or more collection repositories.

* lib/ansible/modules/packaging/os/rhn_channel.py -> https://galaxy.ansible.com/community/general

For further information, please see:
https://github.com/ansible/ansibullbot/blob/master/docs/collection_migration.md
